test/utils/getCalendarMonthWeeks_spec.js

```diff
@@ -38,6 +38,32 @@ describe('getCalendarMonthWeeks', () => {
     });
   });
 
+  describe('padding when enableOutsideDays is false', () => {
+    let weeksWithPadding;
+
+    before(() => {
```

Comment:
`beforeEach` would work too, but here I intended the before all behavior because I'm not mutating this variable. I wrapped in a before block instead of [defining up top like the other variables](https://github.com/moonboots/react-dates/blob/f99af7d544bf83af652b0135fa6b5e16439499e0/test/utils/getCalendarMonthWeeks_spec.js#L8) as a minor perf improvement to avoid running if `.only(...)` was used elsewhere.